options/core/src/main/java/polanski/option/Some.java

```diff
@@ -171,34 +171,34 @@ public OUT call(final IN it) {
             }
         })
                 : first(options).lift(tail(options), new FuncN<OUT>() {
-                    @Override
-                    public OUT call(final Object... list) {
-                        return f.call(combine(mValue, list));
-                    }
-                });
+            @Override
+            public OUT call(final Object... list) {
+                return f.call(combine(mValue, list));
+            }
+        });
```

Comment:
Could the indentation be improved? Looks a bit unnatural to decrease the indentation for the FuncN body